content/guides/index.mdx

```diff
@@ -6,43 +6,43 @@ import {Box, Link, Text} from '@primer/react'
 
 <Box display="grid" gridTemplateColumns={['1fr', null, null, null]} gridGap={4}>
   <div>
-    <Link href="/guides/introduction" sx={{fontWeight: 'bold'}}>
+    <Link href="/design/guides/introduction" sx={{fontWeight: 'bold'}}>
```

Comment:
Is it possible to use the Gatsby `Link` component instead of the current PRC one? The Gatsby one should include the path prefix for you in production.

🔗 [Gatsby docs for Link](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/#in-app-linking)

Reply:
I can do this but wouldn't it be better to change them everywhere? Vs in this fix PR

Reply:
Made a PR to use Gatsby built-in links @rezrah @emilybrick 

https://github.com/primer/design/pull/373